region/norfair/crocomire/Indiana Jones Room.json

```diff
@@ -1522,6 +1583,57 @@
       "flashSuitChecked": true,
       "note": "Use a flash suit to spark horizontally through the speed blocks. Samus may stop above the acid, but it should always be possible to land on one of the platforms."
     },
+    {
+      "link": [2, 5],
+      "name": "No Power Bombs, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {},
+        "comesThroughToilet": "no"
+      },
+      "requires": [
+        "h_RModeCanRefillReserves",
+        {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+        {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+        {"canShineCharge": {"usedTiles": 17, "openEnd": 0}},
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt",
+        {"acidFrames": 30}
```

Comment:
of the 4 strats, this is the only one with acidFrames for interrupting, though, the Mellas sound easier to use

Reply:
Should all have the option of Mella or acid now. Mella may be better at single reserve tank (depending on acid leniency), but there's still a case for acid (4 R-tanks + CF)